tests/monotouch-test/MetalPerformanceShaders/MnistTest.cs

```diff
@@ -42,6 +42,7 @@ public void IsSupported ()
 				TestRuntime.AssertNotSimulator ("Fails with 'Objective-C exception thrown.  Name: NSInvalidArgumentException Reason: -[MTLSimHeap protectionOptions]: unrecognized selector sent to instance 0x600002a09090' - note that we don't call this selector.");
 #endif
 			TestRuntime.IgnoreInCI ("This test seems to make bots keel over and die.");
+			TestRuntime.AssertARM64Desktop (); // Intel Mac is not fast enough.
```

Comment:
Did this happen on the bots? Because the previous line should prevent this test from running on bots at all...

Reply:
No, happened locally, I could repro on my intel Mac, not at all on my m1